src/main/java/seedu/mtracker/ui/TextUi.java

```diff
@@ -3,6 +3,13 @@
 import seedu.mtracker.instrument.Instrument;
 
 public class TextUi {
+    protected static String lineDecorator = "_".repeat(80);
+    private static String logo = "            _________                      __                     \n" +
+            "           |  _   _  |                    [  |  _                 \n" +
+            " _ .--..--.|_/ | | \\_| .--.  ,--.   .---.  | | / ] .---.  _ .--.  \n" +
+            "[ `.-. .-. |   | |  [ `/'`\\]`'_\\ : / /'`\\] | '' < / /__\\\\[ `/'`\\] \n" +
+            " | | | | | |  _| |_  | |    /| | |,| \\__.  | |`\\ \\| \\__., | |     \n" +
+            "[___||__||__]|_____|[___]   \\'-;__/'.___.'[__|  \\_]'.__.'[___]    \n";

```

Comment:
Shift the "+" operator to next line to solve the checkstyle issues 
https://se-education.org/guides/conventions/java/index.html <- look at Place line break to improve readability section

```suggestion
    protected static String lineDecorator = "_".repeat(80);
    private static String logo = "            _________                      __                     \n"
            + "           |  _   _  |                    [  |  _                 \n"
            + " _ .--..--.|_/ | | \\_| .--.  ,--.   .---.  | | / ] .---.  _ .--.  \n"
            + "[ `.-. .-. |   | |  [ `/'`\\]`'_\\ : / /'`\\] | '' < / /__\\\\[ `/'`\\] \n"
            + " | | | | | |  _| |_  | |    /| | |,| \\__.  | |`\\ \\| \\__., | |     \n"
            + "[___||__||__]|_____|[___]   \\'-;__/'.___.'[__|  \\_]'.__.'[___]    \n";

```